src/checkstyle/checks/modifier/ModifierOrderCheck.hx

```diff
@@ -15,7 +15,11 @@ class ModifierOrderCheck extends Check {
 
 	public function new() {
 		super(AST);
+		#if (haxe_ver >= 4.0)
```

Comment:
You could just `.push(FINAL)` in the Haxe 4 branch, instead of copy-pasting the entire thing.

Reply:
Btw, `#if haxe4` may be nicer.

Reply:
is Haxe 5 gonna to set that? :)

Reply:
Yes. Haxe 4 still defines `haxe3`, so it's just a shortcut for `>= 4`.